client/galaxy/scripts/apps/login.js

```diff
@@ -0,0 +1,39 @@
+
+var jQuery = require( 'jquery' ),
+    $ = jQuery,
+    PANEL = require( 'layout/panel' ),
+    _l = require( 'utils/localization' ),
+    PAGE = require( 'layout/page' );
+
+window.app = function app( options, bootstrapped ){
```

Comment:
I mean the increase of boiler plate code, see e.g. the doubling in http://www.mergely.com/4GDDkBHG/ and these two directories: https://github.com/carlfeberhard/galaxy/tree/webpack/client/galaxy/scripts/mvc/app, https://github.com/carlfeberhard/galaxy/tree/webpack/client/galaxy/scripts/apps.
Are they not redundant?